pkg/controller/lifecycle/actuator.go

```diff
@@ -476,7 +478,7 @@ func getSeedResources(oidcReplicas *int32, hibernated bool, namespace string, sh
 		},
 	}
 
-	if err := gutil.InjectGenericKubeconfig(oidcDeployment, shootAccessSecret.Secret.Name); err != nil {
+	if err := gutil.InjectGenericKubeconfig(oidcDeployment, genericKubeconfigName, shootAccessSecret.Secret.Name); err != nil {
```

Comment:
Hey, @rfranzke @timebertt 
Is this good enough for now? Later on when all around the new secrets manager is settled we can sync the Shoot's CA rotation with the oidc webhook's server cert rotation.

Reply:
Yeah, it looks great :)

Reply:
Maybe you can add a fallback to the old, constant name in case the `Cluster` resource is not annotated. This can cover cases where an operator updates the OIDC extension but keeps Gardener running with an older version which does not annotate the `Cluster` resource yet.

Reply:
Isn't this already handled here? https://github.com/gardener/gardener/blob/d67c375eddb407db0c29343916dbaf4320587e03/pkg/extensions/cluster.go#L187

Reply:
Oh yeah, of course 😁 All covered then :)